studio/schemas/documents/specialPages/customerCases.ts

```diff
@@ -0,0 +1,28 @@
+import { defineType } from "sanity";
+
+import { title } from "studio/schemas/fields/text";
+import { titleSlug } from "studio/schemas/schemaTypes/slug";
+
+export const customerCasesID = "customerCases";
+
+const customerCases = defineType({
```

Comment:
I would name this `customerCasesPage` or something, to distinguish it from the actual customer cases document type in sharedStudio.